index.html

```diff
@@ -4,7 +4,128 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
     <title>Portfolio - James Serengia</title>
   </head>
-  <body></body>
+  <body>
+    <header class="header">
+      <div class="header-content row">
+        <div class="logo">Welcome</div>
+        <nav class="nav">
+          <svg
+            class="menu-icon"
+            width="18"
+            height="12"
+            viewBox="0 0 18 12"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              fill-rule="evenodd"
+              clip-rule="evenodd"
+              d="M0 1a1 1 0 0 1 1-1h16a1 1 0 1 1 0 2H1a1 1 0 0 1-1-1zm0 5a1 1 0 0 1 1-1h16a1 1 0 1 1 0 2H1a1 1 0 0 1-1-1zm1 4a1 1 0 1 0 0 2h16a1 1 0 1 0 0-2H1z"
+            />
+          </svg>
+          <ul class="menu-list">
+            <li class="menu-item">
+              <a class="menu-link" href="#portfolio">Portfolio</a>
+            </li>
+            <li class="menu-item">
+              <a class="menu-link" href="#about">About</a>
+            </li>
+            <li class="menu-item">
+              <a class="menu-link" href="#contact">Contact</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <section class="section-hero">
+        <div class="row hero-content">
+          <h1 class="header-text">
+            <span class="part part-1"
+              >Hey there, my name is James Serengia.</span
+            >
+            <span class="part part-2"
+              >A software Engineer based in Nairobi Kenya</span
+            >
+          </h1>
+          <p class="header-description">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <div class="social-icons">
```

Comment:
- [ ] kindly try to fix the alignment of your social icons, they need to be centred in the middle of the screen like shown in the Figma design 

Figma           | Your design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/191589186-b203bc81-54a4-4ceb-abde-c9322e7e141d.png) |![image](https://user-images.githubusercontent.com/89282221/191589084-910ea1fa-3b44-4ae6-8f7b-bad1a54307ce.png)